### ISSUE TITLE
Não reconhece airpod

### ISSUE THREAD

**Issue by wagnercs007** [steam3d/MagicPods-Windows#98]
Programa só fica:  "pesquisando airpods" e não reconhece.

Airpod reconhecido normalmente no iphone.

**Comment by steam3d** [comment]
Hi. Can you upload a log file?

**Comment by wagnercs007** [comment]
ok
[MagicPodsServiceLog.txt](https://github.com/steam3d/MagicPods-Windows/files/6584526/MagicPodsServiceLog.txt)
[MagicPodsUILog.txt](https://github.com/steam3d/MagicPods-Windows/files/6584527/MagicPodsUILog.txt)

**Comment by steam3d** [comment]
Something wrong with your Bluetooth drivers i think. Try to update a bluetooth driver. (Some users have the same issue, they have an Intel Bluetooth and update to newest drivers helped)

**Comment by wagnercs007** [comment]
Sua informação está incorreta.
Drive ja estava funcionando.... e atualizado

Favor dar a informacao correta, o problema persiste.
[Assistente de driver e suporte Intel®.pdf](https://github.com/steam3d/MagicPods-Windows/files/6586492/Assistente.de.driver.e.suporte.Intel.pdf)

**Comment by steam3d** [comment]
I can’t do anything, The MagicPods makes request to Windows. Windows answer with no data. Obviously, if I don't have data, I can't interact with the headphones in any way.

Make sure that you update drivers from web site your vendor. Windows update usually contains not the latest version Bluetooth driver.

**Comment by wagnercs007** [comment]
mas eu te enviei o relatorio tirado do site da propria intel. 
esta tudo atualizado!  
Driver atualizado, windows atualizado... e o airpod reconhece no windows e no iphone, menos no seu programa.

**Comment by steam3d** [comment]
Airpods use both protocol, classic Bluetooth (this protocol you use to listen to music) and Bluetooth low energy.  Some Bluetooth adapters can’t use Classic Bluetooth and Bluetooth Low Energy.

Thus, the problems are either in the driver or in the joy (Bluetooth).

I suggested a possible solution to the problem. The whole Bluetooth stack is controlled by Windows, not me.  For my part, I checked that the program is working normally.
